calc/src/data/moves.ts

```diff
@@ -1349,6 +1354,8 @@ const ADV: {[name: string]: MoveData} = extend(true, {}, GSC, {
   },
 });
 
+delete ADV['Present'];
```

Comment:
Don’t delete the move

Reply:
Present's unusal and useful properties ONLY exist in GSC. Is there any reason to keep it in the calculator in ADV onwards, when the move's damage calculation is changed and no longer useful?

Reply:
Yes, the calc has status moves afterall

Reply:
Alright, removed the delete.